kubernetes_deploy/staging/queue_worker_deployment.yml

```diff
@@ -0,0 +1,62 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  annotations:
+    kubernetes.io/change-cause: "<to be filled in deploy_to_kubernetes script>"
+  name: laa-legal-adviser-api-queue-worker
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        worker: laa-legal-adviser-api-queue-worker
+        env: staging
+        service_area: laa-get-access
+        service_team: cla-fala
+    spec:
+      containers:
+      - image: "<to be set by deploy_to_kubernetes>"
+        name: worker
+        args: ["run_worker"]
```

Comment:
:raising_hand_man: If we change this to `docker/run_worker`, we can remove the extra two lines in the Dockerfile.

Reply:
I've just remove the `args: ["run_worker"]` from the container as we no longer need to overwrite the command in the deployment

Reply:
As this is the worker pod, I'm not sure I follow why.

Reply:
I think we do need the migration to happen as we need django project in a working state but not running.

uwsgi process will run but it will only run the celery app and not the web app configuration.

Reply:
Having written that, I see your point, no need to have the uwsgi process to run. I'll update

Reply:
Yes, you are right, we do need migration. However, if both sets of pods kick off with the migration then we would have concurrent migration running (both deployments would spin off 1-1 new pod concurrently after deployment), which is probably not what we want.